src/__tests__/NewParentDashboard.test.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import NewParentDashboard from '../components/pages/NewParentDashboard/NewParentDashboard';
+import {BrowserRouter as Router} from 'react-router-dom';
+
+
+import configureStore from 'redux-mock-store';
+import { Provider } from 'react-redux';
+
+const mockStore = configureStore([]);
+const store = mockStore();
+
+
+
+
+jest.mock('@okta/okta-react', () => ({
+    useOktaAuth: () => {
+        return {
+            authState: {
+                isAuthenticated: true,
+            },
+            authService: {},
+        };
+    },
+})); 
+
+const ParentComponent = props => {
+    return(
+        <Router>
+            <Provider store = {store}>
+                <NewParentDashboard {...props} />
+            </Provider>
+        </Router>
+    );
+
+};
+
+describe('<NewParentDashboard /> test suite', () => {
+    test('Parent Dashboard', () => {
+        render(<ParentComponent parent={{ children: [] }} />);
+        expect(screen.getByText(/Parent Dashboard/i)).toBeInTheDocument();
+    });
+
+    // test('The new Parent Dashboard renders successfully', () => {
```

Comment:
Why is this commented out? if we are not using it anymore it can be delete. We want to minimize the amount of commented out code we leave in the main repo.

Reply:
I took that comment out and deleted it